airbyte-webapp/src/components/JobItem/components/Logs.tsx

```diff
@@ -34,10 +34,12 @@ const LogsView = styled.div<{ isEmpty?: boolean }>`
 
 type LogsProps = {
   logsArray?: string[];
+  logTimestamp?: number;
 };
 
-const Logs: React.FC<LogsProps> = ({ logsArray }) => {
+const Logs: React.FC<LogsProps> = ({ logsArray, logTimestamp }) => {
   const logsJoin = logsArray?.length ? logsArray.join("\n") : "No logs available";
+  const matchingLineNumbers = getMatchingLineNumbers(logTimestamp, logsArray);
```

Comment:
Since `getMatchingLineNumbers` might take some time to iterate, is this something we may want to wrap in a `useMemo`?